config/manifests/kustomization.yaml

```diff
@@ -0,0 +1,27 @@
+# These resources constitute the fully configured set of manifests
+# used to generate the 'manifests/' directory in a bundle.
+resources:
+- bases/koor-operator.clusterserviceversion.yaml
+- ../default
+- ../samples
+- ../scorecard
+
+# [WEBHOOK] To enable webhooks, uncomment all the sections with [WEBHOOK] prefix.
+# Do NOT uncomment sections with prefix [CERTMANAGER], as OLM does not support cert-manager.
+# These patches remove the unnecessary "cert" volume and its manager container volumeMount.
+patchesJson6902:
+- target:
+    group: apps
+    version: v1
+    kind: Deployment
+    name: controller-manager
+    namespace: system
+  patch: |-
+    # Remove the manager container's "cert" volumeMount, since OLM will create and mount a set of certs.
```

Comment:
So that's the reason why the cert-manager cert stuff in the other files is commented out?

Reply:
Yes, essentially OLM will handle the certificates for us. But I am not quite sure, so I created this issue to ask for clarification: https://github.com/operator-framework/operator-sdk/issues/6257

We can merge this for now so I can test the OLM, and I created an issue to track this on linear: https://linear.app/koorinc/issue/KSD-156/do-we-need-cert-manager